mgc/dscr/oneSample.py

```diff
@@ -0,0 +1,110 @@
+from sklearn.metrics import euclidean_distances
+from ._utils import _CheckInputs
+import numpy as np
+import random
+from .base import DiscriminabilityTest
+from scipy._lib._util import MapWrapper
+
+
+
+class oneSample(DiscriminabilityTest):
+    r"""
+     A class that performs a one-sample test for whether the discriminability differs from random chance,
+     as described in [1]. With :math:`D_X` as the sample discriminability of :math:`X`, it tests whether:
+
+     .. math::
+        H_0: D_X = D_0
+
+     and
+
+     .. math::
+        H_A: D_X > D_0 
+    
+    where :math:`D_0` is the discriminability that would be observed by random chance.
+    
+    References
+    ----------
+    .. [#1Dscr] Eric W. Bridgeford, et al. "Optimal Decisions for Reference 
+                Pipelines and Datasets: Applications in Connectomics." Bioarxiv (2019).
+    """
+
+    def __init__(self):
+        self.stat = None
```

Comment:
I would not define these here

Reply:
I agree this should be in the abstract class